spec/unit/facter/java_version_spec.rb

```diff
@@ -3,6 +3,7 @@
 describe Facter::Util::Fact do
   before {
     Facter.clear
+    Facter.fact(:osfamily).stubs(:value).returns("OpenBSD")
```

Comment:
I think this can be simplified to a separate test:

```ruby
context 'works on OpenBSD' do
      it do
        Facter.fact(:osfamily).stubs(:value).returns("OpenBSD")
        Facter::Util::Resolution.expects(:which).with("java").returns(true)
        Facter::Util::Resolution.expects(:exec).with("java -version 2>&1").returns(java_version_output)
        Facter.fact(:java_version).value.should == "1.7.0_71"
      end
    end
```

Reminds me, I meant to implement OpenBSD for Beaker tests...